src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -64,7 +64,9 @@ const ForgotPassword = ({ intl, dependencies: { dopplerLegacyClient } }) => {
       </Helmet>
       <article className="main-panel">
         <header>
-          <h1 className="logo-doppler-new">Doppler</h1>
+          <h1 className="logo-doppler-new">
+            <a href={_('forgot_password.url_site')}>Doppler</a>
```

Comment:
Don't we need to add target='blank' here?

Reply:
Since this is a simple link can we add this as markup notation? I'm asking out of curiosity, I don't have this issue clear in my mind right now.

Reply:
@cbernat desde Sitios cuando cuando llegas a estos formularios @carodipietro le pidió a Sil que sea todo dentro de la misma ventana para que parezca la misma aplicación. Así que supuse que desde estas pantallas tendría el mismo comportamiento. @carodipietro nos puede confirmar!
@andresmoschini vos sabes algo?

Reply:
Seguramente sea así entonces, en Doppler app era generalmente con blank.

Reply:
@GusBaamonde, tal vez @carodipietro tenga demasiadas notificaciones y no las pueda atender, si podés consultale directamente.

Reply:
@andresmoschini @cbernat recién me contesto Caro me dijo que Ceci tiene razón así que lo cambio ahora

Reply:
good catch @cbernat !